views/timeline.erb

```diff
@@ -27,7 +27,7 @@
   <li><img src="<%= message.author.gravatar(48)%>"><p>
   <strong><a href="<%= message.author.username %>"><%= message.author.username %></a></strong>
   <%= Rack::Utils.escape_html(message.text) %>
-  <small>&mdash; <%= Time.at(message.pub_date) %></small>
+  <small>&mdash; <%= message.created_at %></small>
```

Comment:
Formattet her har _nok_ ændret sig, i forhold til hvordan det var før
Er det noget vi vil gøre noget ved?

Reply:
Turns out, formattet har nok ikke ændret sig alligevel, siden både de vi havde før, og det vi har nu begge smider at Time objekt ud, som så implicit bliver castet til en string for at blive smidt på timeline (Men det ville nok være flot at ændre formattet en gang i fremtiden alligevel 🙂)